diesel_cli/tests/migration_list.rs

```diff
@@ -43,6 +43,22 @@ fn migration_list_lists_pending_applied_migrations() {
     assert!(result.stdout().contains("[X] 12345_create_users_table"));
 }
 
+fn assert_tags_in_order(output: &str, tags: &[&str]) {
+    let mut prev = None;
+    for tag in tags.iter() {
+        let mat = Regex::new(tag).unwrap()
+            .find(&output).expect(&format!("Expected output to contain timestamp: {:?}", tag));
+        match prev {
+            None => prev = Some(mat.0),
+            Some(prev_index) => {
+                let cur_index = mat.0;
+                assert!(prev_index < cur_index);
+                prev = Some(cur_index);
+            }
+        }
+    }
```

Comment:
Using some form of `windows`/`tuple_windows` we can also have a better assertion error message. I think I'll just use `std::slice::windows` for now so we don't need to pull in itertools ([playpen](https://play.integer32.com/?gist=af34bf9c9c509427b413847cb6d3d81a&version=undefined)).

> Is `tag` ever a Regex

Woops, you're right, we can just use `str::find`

> `RegexSet`

That's really useful!